actors/builtin/miner/miner_state_test.go

```diff
@@ -0,0 +1,587 @@
+	assert.NoError(h.t, err)
+}
+
+func (h *minerStateHarness) assertGetSector(rt runtime.Runtime, sectorNo abi.SectorNumber) *miner.SectorOnChainInfo {
+	sectors, found, err := h.s.GetSector(adt.AsStore(rt), sectorNo)
+	assert.NoError(h.t, err)
+	assert.True(h.t, found)
+	assert.NotNil(h.t, sectors)
+	return sectors
+}
+
+// makes a bit field from the passed sector numbers
+func (h *minerStateHarness) assertDeleteSectors(rt runtime.Runtime, sectorNos ...abi.SectorNumber) {
+	bf := newBitFieldFromSectorNos(sectorNos...)
+	err := h.s.DeleteSectors(adt.AsStore(rt), bf)
+	assert.NoError(h.t, err)
+}
+
+//
+// Precommit Store Operations
+//
+
+func (h *minerStateHarness) assertPutPreCommit(rt runtime.Runtime, info *miner.SectorPreCommitOnChainInfo) {
+	err := h.s.PutPrecommittedSector(adt.AsStore(rt), info)
+	assert.NoError(h.t, err)
+}
+
+func (h *minerStateHarness) assertGetPreCommit(rt runtime.Runtime, sectorNo abi.SectorNumber) *miner.SectorPreCommitOnChainInfo {
+	out, found, err := h.s.GetPrecommittedSector(adt.AsStore(rt), sectorNo)
+	assert.NoError(h.t, err)
+	assert.True(h.t, found)
+	return out
+}
+
+func (h *minerStateHarness) assertPreCommitNotFound(rt runtime.Runtime, sectorNo abi.SectorNumber) {
+	out, found, err := h.s.GetPrecommittedSector(adt.AsStore(rt), sectorNo)
+	assert.NoError(h.t, err)
+	assert.False(h.t, found)
+	assert.Equal(h.t, &miner.SectorPreCommitOnChainInfo{}, out)
+}
+
+func (h *minerStateHarness) assertDeletePreCommit(rt runtime.Runtime, sectorNo abi.SectorNumber) {
+	err := h.s.DeletePrecommittedSector(adt.AsStore(rt), sectorNo)
+	assert.NoError(h.t, err)
+}
+
+//
+// Type Construction Methods
+//
+
+// returns a unique SectorPreCommitOnChainInfo with each invocation with SectorNumber set to `sectorNo`.
+func (h *minerStateHarness) newSectorPreCommitOnChainInfo(sectorNo abi.SectorNumber) *miner.SectorPreCommitOnChainInfo {
+	seed := h.getSeed()
+	info := h.newSectorPreCommitInfo(sectorNo)
+	return &miner.SectorPreCommitOnChainInfo{
+		Info:             *info,
+		PreCommitDeposit: abi.NewTokenAmount(int64(seed)),
+		PreCommitEpoch:   abi.ChainEpoch(seed),
+	}
+}
+
+// returns a unique SectorOnChainInfo with each invocation with SectorNumber set to `sectorNo`.
+func (h *minerStateHarness) newSectorOnChainInfo(sectorNo abi.SectorNumber) *miner.SectorOnChainInfo {
+	seed := h.getSeed()
+	info := h.newSectorPreCommitInfo(sectorNo)
+	return &miner.SectorOnChainInfo{
+		Info:            *info,
+		ActivationEpoch: abi.ChainEpoch(seed),
+		DealWeight:      big.NewInt(int64(seed)),
+	}
+}
+
+// returns a unique SectorPreCommitInfo with each invocation with SectorNumber set to `sectorNo`.
+func (h *minerStateHarness) newSectorPreCommitInfo(sectorNo abi.SectorNumber) *miner.SectorPreCommitInfo {
+	seed := h.getSeed()
+	return &miner.SectorPreCommitInfo{
+		RegisteredProof: abi.RegisteredProof_StackedDRG32GiBPoSt,
+		SectorNumber:    sectorNo,
+		SealedCID:       h.cidGetter(),
+		SealRandEpoch:   abi.ChainEpoch(seed),
```

Comment:
Use an explicit value. For the purposes of these tests there's probably no reason not to use a constant.